packages/snaps-controllers/src/interface/SnapInterfaceController.ts

```diff
@@ -93,8 +94,14 @@ export type SnapInterfaceControllerStateChangeEvent =
     SnapInterfaceControllerState
   >;
 
+type NotificationListUpdatedEvent = {
+  type: `${'NotificationServicesController'}:notificationsListUpdated`;
+  payload: [Record<string, any>[]];
```

Comment:
Can we type this? We don't need the specific types from the controller but anything more specific than `any` is preferred.

Reply:
I'm not sure how to type these past `any` since the types themselves for each notification are complex. Any suggestions?